.github/workflows/docs.py

```diff
@@ -0,0 +1,36 @@
+"""Script to generate docs based on given input directory (first argument)
+and output directory (second argument)"""
+import os
+import subprocess
+import sys
+import glob
+import nbformat
+
+code_path = sys.argv[1]
+out_path = sys.argv[2]
+
+os.chdir(code_path)
+os.environ["PDOC_ALLOW_EXEC"] = "1"
```

Comment:
this is only needed for the subprocess, not for current process env, so we can use the `env` parameter of `subprocess.run()`